topics/introduction/tutorials/galaxy-cite/tutorial.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: tutorial_hands_on
+title: Best Practices for Citing Galaxy
+tags:
+  - fair
+  - introduction
+level: Introductory
+questions:
+  - How to cite Galaxy in Publications?
+objectives:
+  - Apply the Galaxy citation guidelines to your own work.
+  - Prepare your methodology section based on your Galaxy Histories.
+time_estimation: 20m
+key_points:
```

Comment:
I would formulate it more like key learnings of the tutorial and less like recommendations. AI suggestion:
- Cite the Galaxy Project: Always reference the primary Galaxy publication.
- Acknowledge the platform: Include the specific Galaxy server you used for your analyses.
- Share your work: Publish your Galaxy Histories and Workflows to make your methods transparent and reproducible.
- Credit your data sources: Cite any public datasets used in your analyses.
- Include tool citations: Export a list of tool citations directly from your Galaxy History.

Not sure if I would use it like this. But maybe a starting point for updating them. :)